### PR TITLE
tidy scripts and dependencies

### PR DESCRIPTION
**Pull request by casperdcl**
- add explicit `keras` dependency
- use `tqdm.keras`
  + update to latest `tqdm`
- some style (`flake8`) fixes

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Awesome, thanks. Did you check that this still matches https://dvc.org/doc/tutorials/versioning by following the tutorial but checking out this branch after `git clone`?

**Comment by casperdcl** [comment]
Yes it's not wip so should be good to ship

**Comment by casperdcl** [comment]
Though I'm still waiting for a PR over at https://github.com/conda-forge/tqdm-feedstock/pulls for conda support (should happen soon). However the PyPI version is live so the example will run fine using `pip install -r requirements.txt` as documented.

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 pillow>=5.3,<6
 scipy>=1.3,<2
 tensorflow>=2,<3
-tqdm>=4,<5
+keras
```
Comment (shcheklein) [inline requirements.txt]: should we fix the major version at least?

Comment (casperdcl) [reply requirements.txt]: Was relying on dependabot :D... also presumably installing at the same time as tensorflow will ensure compatible versions

Comment (shcheklein) [reply requirements.txt]: 👍

**Pull request merged**